pom.xml

```diff
@@ -254,6 +254,7 @@
       -Xep:JUnitValueSource:OFF
       <!-- Until https://github.com/checkstyle/checkstyle/issues/14194. -->
       -Xep:LexicographicalAnnotationListing:OFF
+      -Xep:MockitoMockClassReference:ERROR
```

Comment:
I think there should be a clear separation between OFFs and ERRORs. You added this one right after an OFF with a reason for being OFF. At just a glance, it looks like this new addition is following the reason of the other, but it is not.

I recommend putting all ERRORs on top with no comments, and OFFs one the bottom with comments.

Reply:
Added a commit for these changes!